frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseSlideOutAction.java

```diff
@@ -121,11 +121,11 @@ private void deleteBranchIfRequired(AnActionEvent anActionEvent, String branchNa
     } else if (gitRepository != null) {
       val root = gitRepository.getRoot();
       val configValueOption = getDeleteLocalBranchOnSlideOutGitConfigValue(project, root);
-      if (configValueOption.isEmpty()) {
+      if (configValueOption == null) {
         ModalityUiUtil.invokeLaterIfNeeded(ModalityState.NON_MODAL,
             () -> suggestBranchDeletion(anActionEvent, branchName, gitRepository, project));
-      } else if (configValueOption.isDefined()) {
-        val shouldDelete = configValueOption.get();
+      } else if (configValueOption != null) {
```

Comment:
Nit:
```suggestion
      } else {
```
is probably enough